testimony.yaml

```diff
@@ -23,6 +23,7 @@ CaseComponent:
     - BootdiskPlugin
     - Bootstrap
     - Branding
+    - BVT
```

Comment:
Not necessary to have a BZ component or something in ohsnap related to this testing, its not any particular components testing hence need not to align with DEV processes. Its very QE / Delivery specific.

Since having a component is must in tests and since I cant assign any particular component I had to introduce this new component.

Reporting for these tests would be handled in polarion and RP by creating these 2 new components.